index.html

```diff
@@ -100,7 +100,42 @@ <h2>Get more with low-cost training programs and advanced features.</h2>
 
         <section>
             <h2>Membership</h2>
+            <ul>
+                <li>
+                    <h3>Join for Free</h3>
+                    <p>Reach your fitness goals or maintain your healthy lifestyle with professional training and support from a positive and active online community.</p>
+                    <ul>
+                        <li>~600 Full-Length Workout Videos</li>
+                        <li>Customizable Calendar</li>
+                        <li>Healthy Recipes</li>
+                        <li>Health and Fitness Articles</li>
+                        <li>Possitive and Supportive Online Community</li>
+                        <li>No Creedit Card Needed</li>
+                    </ul>
+                    <a href="#">See More</a>
+                    <img src="https://images.unsplash.com/photo-1560416313-414b33c856a9?ixid=&ixlib=rb-1.2.1&auto=format&fit=crop&w=50" alt="arrow">
```

Comment:
No need to read out "arrow" with a screenreader.

```suggestion
                    <img src="https://images.unsplash.com/photo-1560416313-414b33c856a9?ixid=&ixlib=rb-1.2.1&auto=format&fit=crop&w=50" alt="">
```